common/features/config/networks/static/reducer.ts

```diff
@@ -850,6 +851,31 @@ export const STATIC_NETWORKS_INITIAL_STATE: types.ConfigStaticNetworksState = {
       max: 60,
       initial: 1
     }
+  },
+  WEB: {
+    id: 'WEB',
+    name: 'Webchain',
+    unit: 'WEB',
+    chainId: 24484,
+    isCustom: false,
+    color: '#0092ee',
+    blockExplorer: makeExplorer({
+      name: 'Webchain Explorer',
+      origin: 'https://explorer.webchain.network'
+    }),
+    tokens: [],
+    contracts: [],
+    dPathFormats: {
+      [SecureWalletName.TREZOR]: WEB_DEFAULT,
+      [SecureWalletName.SAFE_T]: WEB_DEFAULT,
+      [SecureWalletName.LEDGER_NANO_S]: WEB_DEFAULT,
```

Comment:
As I mentioned above, please include links to the relevant pull request in third party repositories. It saves time for those reviewing the PR. 

No pull request currently exists for Ledger support. Please remove LEDGER_NANO_S support from this PR until after you can demonstrate Ledger hardware wallet support.

Lucas, you may have heard that I got the Ledger working for WebChain, but we can't include support for it in MyCrypto (nor MEW) until after the pull request has been issued upstream to LedgerHQ.  We still have some work to do before that PR can be made, so LEDGER_NANO_S support should be left out of this until then.